soc/st/stm32/Kconfig.defconfig

```diff
@@ -85,4 +85,12 @@ config FLASH_BASE_ADDRESS
 		if $(DT_FLASH_PARENT_IS_XSPI)
 	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_FLASH))
 
+config STM32_LTDC_FB_SMH_ATTRIBUTE
```

Comment:
Please add a short comment so reader can have an idea of why this is needed (and meaningful to put it here)